pipeline/alert.go

```diff
@@ -671,7 +676,7 @@ func (a *AlertNode) Sensu() *SensuHandler {
 	return sensu
 }
 
-// tick:embedded:AlertNode.Sensu
+// tick:embedded:AlertNode.SensuHandler
```

Comment:
Why was this line changed? This name needs to refer to the method that returns the SensuHandler type not the type itself.